std/path.d

```diff
@@ -2674,10 +2674,12 @@ unittest
     See_Also:
         $(LREF relativePath)
 */
-auto asRelativePath(CaseSensitive cs = CaseSensitive.osDefault, R)
-    (R path, string base)
-    if (isRandomAccessRange!R && isSomeChar!(ElementType!R) ||
-        isNarrowString!R)
+ auto asRelativePath(CaseSensitive cs = CaseSensitive.osDefault, R1, R2)
+    (R1 path, R2 base)
+   if ((isRandomAccessRange!R1 && isSomeChar!(ElementType!R1) ||
+        isNarrowString!R1) &&
+       (isRandomAccessRange!R2 && isSomeChar!(ElementType!R2) ||
+        isNarrowString!R2))
```

Comment:
As far as I can tell, these constraints are for `pathSplitter`. But that has a slightly different constraint:
`isRandomAccessRange!R && hasSlicing!R || isSomeString!R`.
Looks like `hasSlicing` should be in there. I'm not sure if `isNarrowString` vs `isSomeString` makes a difference here.